queries.sql

```diff
@@ -43,4 +43,12 @@ select COUNT(*) from animals where escape_attempts = 0
 SELECT AVG(weight_kg) FROM animals;
 SELECT neutered as neutered_animals , COUNT(*) as number_of_escape FROM animals WHERE escape_attempts > 0 GROUP BY neutered;
 SELECT species, MIN(weight_kg) as minimum_weight , MAX(weight_kg) as maximum_weight FROM animals GROUP BY species;
-SELECT species, AVG(escape_attempts) as avg_number_of_escape FROM animals WHERE date_of_birth BETWEEN '990-01-01' AND '2000-12-31' GROUP BY species;
\ No newline at end of file
+SELECT species, AVG(escape_attempts) as avg_number_of_escape FROM animals WHERE date_of_birth BETWEEN '990-01-01' AND '2000-12-31' GROUP BY species;
```

Comment:
- [ ] you dropped the species from the animals' table and in this query, you are trying to select species from animals which returns an error also there is a typo in the date so please make sure to fix it so that it works

Reply:
Please the line you highlighted was from a previous task. This is totally different from what I worked on today